canonical-web-and-design/snapcraft.io
# Dispute reserved name

Fixes https://github.com/canonical-web-and-design/snapcraft.io/issues/1885

## QA
- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/register-snap
- Try and register "bitcoin-test"
- Error should show saying the name is reserved
- Click the link in the notification
- Add some details
- SUCCESS
- High five the nearest person
